src/index.js

```diff
@@ -310,15 +310,23 @@ module.exports = React.createClass({
    * Scroll by index
    * @param  {number} index offset index
    */
-  scrollTo(index) {
+  scrollTo(index, animated = true) {
     if (this.state.isScrolling || this.state.total < 2) return
     let state = this.state
     let diff = (this.props.loop ? 1 : 0) + index + this.state.index
     let x = 0
     let y = 0
     if(state.dir == 'x') x = diff * state.width
     if(state.dir == 'y') y = diff * state.height
-    this.refs.scrollView && this.refs.scrollView.scrollTo(y, x)
+    if (this.refs.scrollView) {
+      if (Platform.OS === 'ios') {
+        this.refs.scrollView.scrollTo({y, x, animated})
+      } else {
```

Comment:
Perhaps use an explicit Android check here, given that Tizen and Windows support is coming?